### ISSUE TITLE
Registration error being treated as server error

### ISSUE THREAD

**Issue by Tam** [jamesedmonston/graphql-authentication#74]
If a user tries to register with an email that already exists, the below is returned (with a 200 status code):

```json
{
  "errors": [
    {
      "debugMessage": "{\"email\":[\"Email \\\"test@example.com\\\" has already been taken.\"]}",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "trace": [
        {
          "file": "/var/www/html/vendor/jamesedmonston/graphql-authentication/src/services/UserService.php",
          "line": 523,
          "call": "jamesedmonston\\graphqlauthentication\\services\\ErrorService::throw('{\"email\":[\"Email \\\"test@example.com\\\" has already been taken.\"]}')"
        },
        {
          "file": "/var/www/html/vendor/jamesedmonston/graphql-authentication/src/services/UserService.php",
          "line": 194,
          "call": "jamesedmonston\\graphqlauthentication\\services\\UserService::create(array(7), 1)"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 624,
          "call": "jamesedmonston\\graphqlauthentication\\services\\UserService::jamesedmonston\\graphqlauthentication\\services\\{closure}(null, array(7), array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 550,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveFieldValueOrError(instance of GraphQL\\Type\\Definition\\FieldDefinition, instance of GraphQL\\Language\\AST\\FieldNode, instance of Closure, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 474,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Mutation, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 858,
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'register')"
        },
        {
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'register')"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 860,
          "function": "array_reduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 490,
          "call": "GraphQL\\Executor\\ReferenceExecutor::promiseReduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 263,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFieldsSerially(GraphQLType: Mutation, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, array(2), array(7), 'Join', null)"
        },
        {
          "file": "/var/www/html/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'mutation Join($email: String!, $password: String!, $firstName: String!, $lastName: String!, $website: String, $organisation: String, $acceptTerms: Boolean) {\n  register(\n    email: $email\n    password: $password\n    firstName: $firstName\n    lastName: $lastName\n    website: $website\n    organisation: $organisation\n    acceptTerms: $acceptTerms\n  ) {\n    jwt\n    refreshToken\n    __typename\n  }\n}\n', null, array(2), array(7), 'Join', null, array(26))"
        },
        {
          "file": "/var/www/html/vendor/craftcms/cms/src/services/Gql.php",
          "line": 544,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'mutation Join($email: String!, $password: String!, $firstName: String!, $lastName: String!, $website: String, $organisation: String, $acceptTerms: Boolean) {\n  register(\n    email: $email\n    password: $password\n    firstName: $firstName\n    lastName: $lastName\n    website: $website\n    organisation: $organisation\n    acceptTerms: $acceptTerms\n  ) {\n    jwt\n    refreshToken\n    __typename\n  }\n}\n', null, array(2), array(7), 'Join', null, array(26))"
        },
        {
          "file": "/var/www/html/vendor/craftcms/cms/src/controllers/GraphqlController.php",
          "line": 172,
          "call": "craft\\services\\Gql::executeQuery(instance of craft\\models\\GqlSchema, 'mutation Join($email: String!, $password: String!, $firstName: String!, $lastName: String!, $website: String, $organisation: String, $acceptTerms: Boolean) {\n  register(\n    email: $email\n    password: $password\n    firstName: $firstName\n    lastName: $lastName\n    website: $website\n    organisation: $organisation\n    acceptTerms: $acceptTerms\n  ) {\n    jwt\n    refreshToken\n    __typename\n  }\n}\n', array(7), 'Join', true)"
        },
        {
          "call": "craft\\controllers\\GraphqlController::actionApi()"
        },
        {
          "file": "/var/www/html/vendor/yiisoft/yii2/base/InlineAction.php",
          "line": 57,
          "function": "call_user_func_array(array(2), array(0))"
        },
        {
          "file": "/var/www/html/vendor/yiisoft/yii2/base/Controller.php",
          "line": 181,
          "call": "yii\\base\\InlineAction::runWithParams(array(0))"
        },
        {
          "file": "/var/www/html/vendor/craftcms/cms/src/web/Controller.php",
          "line": 189,
          "call": "yii\\base\\Controller::runAction('api', array(0))"
        },
        {
          "file": "/var/www/html/vendor/yiisoft/yii2/base/Module.php",
          "line": 534,
          "call": "craft\\web\\Controller::runAction('api', array(0))"
        },
        {
          "file": "/var/www/html/vendor/craftcms/cms/src/web/Application.php",
          "line": 277,
          "call": "yii\\base\\Module::runAction('graphql/api', array(0))"
        },
        {
          "file": "/var/www/html/vendor/yiisoft/yii2/web/Application.php",
          "line": 104,
          "call": "craft\\web\\Application::runAction('graphql/api', array(0))"
        },
        {
          "file": "/var/www/html/vendor/craftcms/cms/src/web/Application.php",
          "line": 262,
          "call": "yii\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/var/www/html/vendor/yiisoft/yii2/base/Application.php",
          "line": 392,
          "call": "craft\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/var/www/html/web/index.php",
          "line": 35,
          "call": "yii\\base\\Application::run()"
        }
      ]
    }
  ]
}
```

**Comment by mgambella** [comment]
@jamesedmonston the same happens also on login:

```
{
  "errors": [
    {
      "debugMessage": "We couldn't log you in with the provided details",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "trace": [
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/jamesedmonston/graphql-authentication/src/services/UserService.php",
          "line": 148,
          "call": "jamesedmonston\\graphqlauthentication\\services\\ErrorService::throw('We couldn\\'t log you in with the provided details')"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 624,
          "call": "jamesedmonston\\graphqlauthentication\\services\\UserService::jamesedmonston\\graphqlauthentication\\services\\{closure}(null, array(2), array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 550,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveFieldValueOrError(instance of GraphQL\\Type\\Definition\\FieldDefinition, instance of GraphQL\\Language\\AST\\FieldNode, instance of Closure, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 474,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Mutation, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 858,
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'authenticate')"
        },
        {
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'authenticate')"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 860,
          "function": "array_reduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 490,
          "call": "GraphQL\\Executor\\ReferenceExecutor::promiseReduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 263,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFieldsSerially(GraphQLType: Mutation, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, array(2), array(2), 'Authenticate', null)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', null, array(2), array(2), 'Authenticate', null, array(26))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/craftcms/cms/src/services/Gql.php",
          "line": 544,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', null, array(2), array(2), 'Authenticate', null, array(26))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/craftcms/cms/src/controllers/GraphqlController.php",
          "line": 172,
          "call": "craft\\services\\Gql::executeQuery(instance of craft\\models\\GqlSchema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', array(2), 'Authenticate', true)"
        },
        {
          "call": "craft\\controllers\\GraphqlController::actionApi()"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/yiisoft/yii2/base/InlineAction.php",
          "line": 57,
          "function": "call_user_func_array(array(2), array(0))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/yiisoft/yii2/base/Controller.php",
          "line": 181,
          "call": "yii\\base\\InlineAction::runWithParams(array(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/yiisoft/yii2/base/Module.php",
          "line": 534,
          "call": "yii\\base\\Controller::runAction('api', array(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 287,
          "call": "yii\\base\\Module::runAction('graphql/api', array(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 596,
          "call": "craft\\web\\Application::runAction('graphql/api', array(1))"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 266,
          "call": "craft\\web\\Application::_processActionRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/vendor/yiisoft/yii2/base/Application.php",
          "line": 392,
          "call": "craft\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/Users/michele/Data/Development/Freelancing/Bezar/AWV/Code/louver-cms/web/index.php",
          "line": 21,
          "call": "yii\\base\\Application::run()"
        }
      ]
    }
  ]
}
```

**Comment by mgambella** [comment]
@jamesedmonston the same happens also on login or when an user tries to login without having activated the account before.

**Comment by mgambella** [comment]
@jamesedmonston here is the stack trace for an attempted login when in dev mode

```
{
  "errors": [
    {
      "debugMessage": "We couldn't log you in with the provided details",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "trace": [
        {
          "file": "/www/cms/vendor/jamesedmonston/graphql-authentication/src/services/UserService.php",
          "line": 148,
          "call": "jamesedmonston\\graphqlauthentication\\services\\ErrorService::throw('We couldn\\'t log you in with the provided details')"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 624,
          "call": "jamesedmonston\\graphqlauthentication\\services\\UserService::jamesedmonston\\graphqlauthentication\\services\\{closure}(null, array(2), array(2), instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 550,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveFieldValueOrError(instance of GraphQL\\Type\\Definition\\FieldDefinition, instance of GraphQL\\Language\\AST\\FieldNode, instance of Closure, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 474,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Mutation, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 858,
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'authenticate')"
        },
        {
          "call": "GraphQL\\Executor\\ReferenceExecutor::GraphQL\\Executor\\{closure}(array(0), 'authenticate')"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 860,
          "function": "array_reduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 490,
          "call": "GraphQL\\Executor\\ReferenceExecutor::promiseReduce(array(1), instance of Closure, array(0))"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 263,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFieldsSerially(GraphQLType: Mutation, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, array(2), array(2), 'Authenticate', null)"
        },
        {
          "file": "/www/cms/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', null, array(2), array(2), 'Authenticate', null, array(26))"
        },
        {
          "file": "/www/cms/vendor/craftcms/cms/src/services/Gql.php",
          "line": 544,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', null, array(2), array(2), 'Authenticate', null, array(26))"
        },
        {
          "file": "/www/cms/vendor/craftcms/cms/src/controllers/GraphqlController.php",
          "line": 172,
          "call": "craft\\services\\Gql::executeQuery(instance of craft\\models\\GqlSchema, 'mutation Authenticate($email: String!, $password: String!) {\n  authenticate(email: $email, password: $password) {\n    jwt\n    jwtExpiresAt\n    refreshToken\n    refreshTokenExpiresAt\n    user {\n      ... on User {\n        id\n        email\n        status\n        username\n        firstName\n        lastName\n      }\n    }\n  }\n}\n', array(2), 'Authenticate', true)"
        },
        {
          "call": "craft\\controllers\\GraphqlController::actionApi()"
        },
        {
          "file": "/www/cms/vendor/yiisoft/yii2/base/InlineAction.php",
          "line": 57,
          "function": "call_user_func_array(array(2), array(0))"
        },
        {
          "file": "/www/cms/vendor/yiisoft/yii2/base/Controller.php",
          "line": 181,
          "call": "yii\\base\\InlineAction::runWithParams(array(1))"
        },
        {
          "file": "/www/cms/vendor/yiisoft/yii2/base/Module.php",
          "line": 534,
          "call": "yii\\base\\Controller::runAction('api', array(1))"
        },
        {
          "file": "/www/cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 287,
          "call": "yii\\base\\Module::runAction('graphql/api', array(1))"
        },
        {
          "file": "/www/cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 596,
          "call": "craft\\web\\Application::runAction('graphql/api', array(1))"
        },
        {
          "file": "/www/cms/vendor/craftcms/cms/src/web/Application.php",
          "line": 266,
          "call": "craft\\web\\Application::_processActionRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/www/cms/vendor/yiisoft/yii2/base/Application.php",
          "line": 392,
          "call": "craft\\web\\Application::handleRequest(instance of craft\\web\\Request)"
        },
        {
          "file": "/www/cms/web/index.php",
          "line": 21,
          "call": "yii\\base\\Application::run()"
        }
      ]
    }
  ]
}
```

**Comment by mgambella** [comment]
I found out the the something is wrong in the **1.11.5** version because it is working again after I downgraded the plugin to **1.11.4** in composer.json

**Comment by jamesedmonston** [comment]
Have pushed a fix for this. You'll now see a contextual error such as:

```json
{
  "errors": [
    {
      "message": "Username \"your@email.com\" has already been taken.",
      "extensions": {
        "code": "INVALID",
        "category": "graphql"
      },
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "register"
      ]
    }
  ]
}
```

**Comment by jamesedmonston** [comment]
@Tam @howells @mgambella I've just released `1.12.0` with this! 👍
